src/pages/onboarding/i18n.js

```diff
@@ -24,7 +24,10 @@ export const tutorialPanelI18nStrings = {
 export const overlayI18nStrings = {
   stepCounterText: (stepIndex, totalStepCount) => `Step ${stepIndex + 1}/${totalStepCount}`,
   taskTitle: (taskIndex, taskTitle) => `Task ${taskIndex + 1}: ${taskTitle}`,
-  labelHotspot: openState => (openState ? 'hide annotation' : 'show annotation'),
+  labelHotspot: (openState, stepIndex, totalStepCount) =>
+  openState
+    ? `close annotation for step ${stepIndex + 1}/${totalStepCount}`
```

Comment:
this gets announced (in VoiceOver at least) as e.g. "one slash three", which seems like it's not entirely clear. Would it be better to write it out fully as e.g. "{step} of {total}" instead?

Reply:
It is good point. Will update it in next revision.